packages/mobile/e2e/init.js

```diff
@@ -1,11 +1,11 @@
+const detox = require('detox')
 import { setDemoMode } from './src/utils/utils'
 
 beforeAll(async () => {
   await device.launchApp({
+    newInstance: false,
     permissions: { notifications: 'YES', contacts: 'YES' },
   })
-})
-
-beforeEach(async () => {
+  await device.setURLBlacklist(['.*blockchain-api-dot-celo-mobile-alfajores.appspot.com.*'])
```

Comment:
What does this do? could be nice to add a small comment explaining it.

Reply:
This prevents detox sync form waiting on requests to the listed API. This API is called every time the user lands on the transactions home screen, so removing the waiting increases the speed of the tests.